data-prepper-plugins/aws-plugin/src/test/java/org/opensearch/dataprepper/plugins/aws/AwsSecretManagerConfigurationTest.java

```diff
@@ -41,6 +43,11 @@
 @ExtendWith(MockitoExtension.class)
 class AwsSecretManagerConfigurationTest {
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper(new YAMLFactory());
+
+    static {
```

Comment:
At this point, it is probably better to just make `OBJECT_MAPPER` an instance variable.